src/components/baseComponents/homeButton/HomeButton.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import HomeButton from './HomeButton';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+describe("Home Button", () => {
+  it("should display a Home Button", () => {
+    const history = createMemoryHistory();
+    
+    const { container } = render(
+      <Router history={history}>
+        <HomeButton link={"/test"} />
+      </Router>
+    );
+
+    expect(container.firstChild).toHaveClass('home-button-container');
```

Comment:
Try to use things like aria-label rather than classes for testing - the name of the class might change yet but the functionality would stay the same.

Reply:
Ok